connector-packager/tests/test_xsd_validator.py

```diff
@@ -158,7 +172,7 @@ def test_warn_authentication_attribute(self):
             logging.getLogger('packager_logger').warning('dummy message')
             warn_file_specific_rules(file_to_test, test_tdr_file)
 
-        self.assertEqual(len(cm.output), 1)
+        self.assertEqual(len(cm.output), 2)
```

Comment:
Why did these tests change?

Reply:
The error messages include the warning for dbname.